### PR TITLE
New Habits App - Nathalie Nordström

### PR DESCRIPTION
**Pull request by nathalienordstrom**
My reflections on how this week's project turned out:

I think it's cool to be able to create an app, the amount of new information meant that I could not go all the way with my idea. I'm really proud of what I've done, and I will leave it now when it's not broken and the next time push it farther.

https://snack.expo.io/@nathalie-nordstrom/unnamed-snack

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,26 +1,33 @@
 import React from 'react'
 import styled from 'styled-components/native'
+import  HomePage from './components/HomePage'
+import  Habits from './Habits'
```
Comment (EmOlsz) [inline App.js:4]: Shouldn't `Habits` be also in `/components` folder?

File: App.js [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
-    <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
+    <NavigationContainer>
+      <Stack.Navigator>
+        <Stack.Screen 
+        name="Home" 
+        component={HomePage}
```
Comment (EmOlsz) [inline App.js:19]: Some wrong indentations

File: Habits.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+`
+const List = styled.View`
+margin-top: 20px;
+`
+
+const Habits = (navigation, route ) => {
+  const [todos, setTodos] = useState([
+    // { text: 'think of 5 things you are grateful for', key: ' 1 ' },
+    // { text: 'breath deep, 10 times', key: ' 2 ' },
+    // { text: 'write in your diary', key: ' 3 ' },
```
Comment (EmOlsz) [inline Habits.js:25]: Let's delete this commented code

File: components/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    const navigateToHabits = () => {
+
+        navigation.navigate("Habits")
+    };
+
+    return (
+        <HomeContainer source={headerImage}>
+            <Header>
+
+            </Header>
```
Comment (EmOlsz) [inline components/HomePage.js:56]: I don't think you need empty tag

File: components/ToDoItem.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { TouchableOpacity } from 'react-native'
+import styled from 'styled-components/native';
+
+
+const ToDos = styled.Text`
+
+
+`
```
Comment (EmOlsz) [inline components/ToDoItem.js:9]: This is styled component without styles - we don't need it :)

File: components/ToDoItem.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+margin-top: 16px;
+`
+
+
+ const ToDoItem = ({ item, pressHandler }) => {
+
+    return (
+        <TouchableOpacity onPress={() => {
+            pressHandler(item.key)
+        }}
```
Comment (EmOlsz) [inline components/ToDoItem.js:23]: We don't have to break line here, this is completely enough :

```
<TouchableOpacity onPress={() =>  pressHandler(item.key)}>
```

**Review by EmOlsz — CHANGES_REQUESTED** [review]
Hey Nathalie! Let's fix issues pointed out in my comments, after that I'll be able to approve your project :)

File: components/HomePage.js [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+            <TextTwo>
+            With New Habits you can make life
+            as you want it!
+            </TextTwo>
+            </TextContainer>
+            <Footer>
+            <Button title="New Habits >" onPress={navigateToHabits}> </Button>
+            </Footer>
+
+        </HomeContainer>
```
Comment (HIPPIEKICK) [inline components/HomePage.js:35]: Nice job with the changes Nathalie! There are still some places that have a weird indentation, such as here 👆. Also, remember to indent styled components, e.g:
```
const Main = styled.View`
  flex:1;
  margin-bottom: 70px;
`
```
Also, remove unnecessary empty lines, and make sure you are consistent regarding semicolons.

Comment (HIPPIEKICK) [reply components/HomePage.js]: This still applies 👆

File: App.js [hunk 8]
```diff
@@ -1,26 +1,28 @@
...
+          name="Home"
+          component={HomePage}
+        />
+        <Stack.Screen
+          name="Habits"
+          component={Habits} />
+      </Stack.Navigator>
+    </NavigationContainer>
+
   )
```
Comment (HIPPIEKICK) [inline App.js:25]: Missing semicolon in all files after `return ()`

File: components/Add.js [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+                </TouchableOpacity>
+            </ButtonBox>
+        </Main>
+    )
+};
+
+const Main = styled.View`
+    flex: 1;
+    margin-bottom: 70px;
+`
```
Comment (HIPPIEKICK) [inline components/Add.js:32]: Missing semi colon after all styled components in all files

**Review by HIPPIEKICK — APPROVED** [review]

**Pull request closed without merging**